src/routes/LandingPage.tsx

```diff
@@ -1,56 +1,110 @@
-                    projections
-                </Text>
-                <Button variant={"solid"} style = {{marginTop: '200px'}} onClick={(_ => console.log("clicked!"))}> Register Now! </Button>
-                <div style={{ border: '1px solid white', padding: '20px', width: '300px', margin: '20px auto' }}>
-                    <Box
-                        borderWidth="1px"
-                        borderRadius="lg"
-                        padding="4"
-                        width="400px"
-                        height = "400px"
-                        margin="auto" 
-                        marginTop="100px" 
-                        marginBottom="100px" 
-                        display="flex"
-                        flexDirection="column"
-                        justifyContent="center"
-                        alignItems="center"
-            >   
-                     <Text fontSize="xl" fontWeight="bold" marginBottom="2">Speaker Series</Text>
-                
-                    </Box>
-                    
-                </div>
-            </div>
-        </>
-    );
-}
+import { Text, Center, Box, Button, HStack, SimpleGrid, Image, VStack} from "@chakra-ui/react";
+import { pictureBox, poloroidBox, jeapordyBox, sponsorBox, bigBox, smallBox} from "../customTheme";
+export function LandingPage() {
+	return (
+	  <>
+		<Box bg="">
+		  <Text variant="baseStyle">
+        <Center h='300px'>
+          <Text>Reflections | Projections </Text>
+        </Center>
+        <Text>September 18 - 22</Text>
+        <VStack>
+          <Button variant={"solid"}>LOGIN</Button>
+          <Button variant={"solid"}>REGISTER</Button>
+        </VStack>
+		  </Text>
+		</Box>
+        {}
 
+    <SimpleGrid
+      h='800px'
+      w='900px'
+      columns={2}
+      spacing={4}
+      margin='auto'
+      marginTop='20vh'
+      bg='black'
+  >
+    <Box bg='gray' height='100%' />
+    <Box bg='gray' height='100%' />
+    <Box bg='gray' height='100%' />
+    <Box bg='gray' height='100%' />
+  </SimpleGrid>
+
+    {/* insert poloroid pictures here */}
+    <HStack> 
+      <Box style = {poloroidBox}> 
+      </Box> 
+      <Box style = {poloroidBox}> 
+      </Box> 
+      <Box style = {poloroidBox}> 
+      </Box> 
+      <Box style = {poloroidBox}> 
+      </Box> 
+      <Box style = {poloroidBox}> 
+      </Box> 
+    </HStack> 
+
+    <Center height="100vh"> 
+      <HStack> 
+        <Box style = {pictureBox} transform = "rotate(-20deg)"> 
+          <Image/>{/*insert picture here*/}
+        </Box> 
+        <Box style = {pictureBox} transform = "rotate(20deg)" marginLeft = "-50px"> 
+        < Image/>{/*insert picture here*/}
+        </Box> 
+        <Box style = {pictureBox} transform = "rotate(-10deg)" marginLeft = "-50px" > 
+          <Image/>{/*insert picture here*/}
+        </Box> 
+      </HStack>
+     
+    </Center>
+    <Center height = "50vh"> 
+      <Button variant={"solid"}>FAQ</Button>
+    </Center> 
```

Comment:
this should just be a text field! the grid area below it is the FAQ, and we just need a title for it